django_tasks/backends/immediate.py

```diff
@@ -37,6 +38,8 @@ def enqueue(
             task=task,
             id=str(uuid4()),
             status=status,
+            enqueued_at=timezone.now(),
```

Comment:
Issue: `enqueued_at` should be calculated from before the task was run, rather than afterwards.

Reply:
Absolutely. Addressed in the current squash & rebase.